### PR TITLE
Flake8

### PR DESCRIPTION
**Pull request by kalbermattenm**
This PR adds support for using flake8 on the Python code.

@voisardf, maybe something for CRDPPF
@monodo, well, well, well... Could you review that... ?

### CODE REVIEW

**Comment by kalbermattenm** [comment]
@monodo, could you review that... ?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,42 @@
...
+    Drive:
+    cd where_ever\historic_cadastre
+    git init
+    git remote add upstream https://github.com/sitn/historic_cadastre.git
+    git fetch upstream
+    git merge upstream/master
+
+Next, run bootstrap (we are assuming that Python is in our path):
+
+    python bootstrap-buildout.py --allow-site-packages
```
Comment (monodo) [inline README.md]: missing bootstrap file

Comment (monodo) [inline README.md]: Was not the plan to replace buildout by make ?

Comment (kalbermattenm) [inline README.md]: Yes... sorry... I did not re-read this file... This PR is quite old... I will refactor this file as well...

**Comment by kalbermattenm** [comment]
Replaced by #56

**Pull request closed without merging**